lib/recurly/client.rb

```diff
@@ -260,19 +259,13 @@ def set_api_key(api_key)
     end
 
     def scope_by_site(path, **options)
-      if site = site_id || options[:site_id]
+      if (site = site_id || options[:site_id]) && !path.start_with?("/sites/#{site}")
```

Comment:
Perhaps let's add a comment about what this is doing but that it's not officially supported.